util.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * Utility functions for use with Stencila Schema in Typescript and Javascript
+ */
+import Ajv from 'ajv'
+import betterAjvErrors from 'better-ajv-errors'
+import fs from 'fs-extra'
+import globby from 'globby'
+import path from 'path'
+import * as stencila from './types'
+
+/**
+ * Create a node of a type
+ * @param type The name of the type
+ * @param initial Initial values for properties
+ * @param validation Should validation be done?
+ */
+export function create<Key extends keyof stencila.Types>(
+  type: Key,
+  initial: { [key: string]: any } = {},
+  validation: boolean = true
+): stencila.Types[Key] {
+  const node = { type, ...initial }
+  if (validation) validate(node, type)
+  return node as stencila.Types[Key]
```

Comment:
In general, instead of casting using `as` I'd recommend declaring the `node` intended type on line 22.
Not really a problem here, but casting can cause us to lose a lot of type safety.

``` ts
  const node: stencila.Types[Key] = { type, ...initial }
  if (validation) validate(node, type)
  return node
```

The result is the same, but might help us catch any unlawful operations on `node` down the road.

Reply:
Thanks, the `as` wasn't necessary and is now removed.